vendor/bat-native-ads/src/bat/ads/internal/bundle_state.cc

```diff
@@ -0,0 +1,17 @@
+// /* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
Should it be 2020 here? Also seems like we're accidentally using both types of comment symbols in this comment header.

Reply:
Fixed